.gitignore

```diff
@@ -116,3 +116,5 @@ app.*.symbols
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 !/dev/ci/**/Gemfile.lock
+/packages/flutter/example/linux/flutter/ephemeral/
```

Comment:
this should not be needed, we already have a gitignore entry for these files on master